modules/ROOT/pages/hosting-options.adoc

```diff
@@ -105,6 +105,24 @@ Replace `<url>` with a valid URL that includes the DLB name and domain (gathered
 ....
 datagraph load-balancer-config add examplelb.lb.anypointdns.net/datagraph/graphql
 ....
++
+For example, consider the following mapping rules for `examplelb.lb.anypointdns.net` depending on the value of *Input Path*:
++
+[%header%autowidth.spread]
+|===
+|Index |Input URI (Input Path) |App name |App URI (Output Path)
+|0 |`/graphql` |`examplelb.lb.anypointdns.net` |`/graphql`
```

Comment:
Hi @IsaacEldridge , It looks like there is a misunderstanding with the domain (App name) of a datagraph app, they are something like this: datagraph-xz082edx-1585-901xe374-d0x0.  (that is datagraph-{part of the orgId}-{part of the envId})
If we use something like ```lb.anypointdns.net``` or ```examplelb.lb.anypointdns.net``` it might be confusing with the final DLB url.

Having that into account, I would change the line 69, and the following uses of the datagraph app, something like this: datagraph-xz082edx-1585-901xe374-d0x0


So the table will looks like (I added the protocol column, because it is important):

|0 |`/graphql` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql`|https
|1 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql`|https
|2 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/`|https
|3 |`/{app}/` |`{app}` | `/`|https

Reply:
I made some changes.